runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -66,7 +66,7 @@
      * A view of streams in the Corfu server instance.
      */
     @Getter(lazy = true)
-    private final MultiStreamView multiStreamsView = new MultiStreamView(this);
+    private final StreamsView streams = new StreamsView(this);
```

Comment:
I think if you're going to do this, you should get rid of the get prefix on the autogenerated getter (no need to make it lazy, that was a remenant if the past...)